### PR TITLE
Fix markdown instructions in Star Wars

### PR DESCRIPTION
**Pull request by Hamms**
The invalid markdown format we use here was invalidated in
https://github.com/code-dot-org/code-dot-org/pull/27120 and wasn't
caught by our "markdown conversion" project because it's in levels.js
rather than levelbuilder

### CODE REVIEW

File: apps/src/studio/levels.js [hunk 1]
```diff
@@ -3555,44 +3555,47 @@ var js_hoc2015_event_free_markdown = [
   '',
   '<details class="hoc2015">',
   '<summary>Example project ideas</summary>',
-  '<p>**Example 1**',
+  '<p><strong>Example 1</strong>',
```
Comment (joshlory) [inline apps/src/studio/levels.js:3558]: Markdown inside HTML isn't allowed with the new parser?

**Review by joshlory — APPROVED** [review]
LGTM. Are there other level.js instructions we should preemptively check?

**Comment by joshlory** [comment]
Was this caught by Eyes tests, a user report, or something else?

File: apps/src/studio/levels.js (hunk 1 above)
Comment (Hamms) [reply apps/src/studio/levels.js:3558]: correct. Specifically, content that should be parsed as HTML must be separated from content that should be parsed as Markdown with blank lines

See https://daringfireball.net/projects/markdown/syntax#html

**Comment by Hamms** [comment]
This was caught by me; an eyes test failed on this level for unrelated reasons, and I noticed this problem while looking at the level

**Comment by Hamms** [comment]
It would probably be worth spot-checking the other levels.js instructions; I think most of them tend to be pretty simple, though, so I wouldn't expect there to be similar violations

**Pull request merged**